app/build.gradle

```diff
@@ -94,8 +94,9 @@ dependencies {
     implementation 'com.squareup.retrofit2:converter-gson:2.5.0'
     implementation 'com.squareup.retrofit2:adapter-rxjava2:2.5.0'
     implementation 'com.squareup.picasso:picasso:2.5.2'
-    implementation 'com.squareup.okhttp3:okhttp:3.12.0'
-    implementation 'com.squareup.okhttp3:logging-interceptor:3.12.0'
+    implementation 'com.squareup.okhttp3:okhttp:3.13.1'
+    implementation 'com.squareup.okhttp3:logging-interceptor:3.13.1'
+    implementation "org.conscrypt:conscrypt-android:2.0.0"
```

Comment:
Did you measure the size change already?

Reply:
Size of Tusky 5.1 apk: 3.749kB
size apk with Conscrypt:  6.986kB (download size will be much smaller for Google Play users because of App Bundles)

Reply:
While it's 2x increase, I don't think many people will notice because it's still quite small.